File: Data Structures & Algorithms/reverse-string/submission-2.js
class Solution {
    reverseString(s) {
        let st = 0 ;
        let end = s.length -1;

        while(st < end){
           [s[st] , s[end]] = [s[end] , s[st]]
            st++;
            end--;
        }
    }
}

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-2.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int bestBuy = prices[0];
        int maxPf = 0;

        for (int i = 1; i < prices.size(); i++) {
            maxPf = max(maxPf, prices[i] - bestBuy);
            bestBuy = min(bestBuy, prices[i]);
        }

        return maxPf;
    }
};